davenjeru/django-getting-started
# Set up polls app

**What does this PR do?**
Initiates the Polls app.

**How can this be manually tested?**
Clone [this](https://github.com/davenjeru/django-getting-started/tree/set-up-polls-app) repository and initiate a virtual environment for the project like so:
`virtualenv venv`
`source venv/bin/activate`

Install the requirements from the requirements.txt file like so:
`pip install -r requirements.txt`

Start the Django development server like so:
`python manage.py runserver`

When you visit `/polls/` on the browser, you should see a message like this:
 _**Hello, world. You're at the polls index.**_
